packages/application-components/src/theming.ts

```diff
@@ -45,6 +45,7 @@ export const themesOverrides = {
     marginForPageContent: `${uiKitDesignTokens.spacing50} ${appKitSpacing55}`,
     marginTopForPageSubtitle: uiKitDesignTokens.spacing20,
     marginTopForTabControls: uiKitDesignTokens.spacing40,
+    paddingForAppbar: `0 ${appKitSpacing55}`,
```

Comment:
Since we already decided to remove these kind of design tokens, I think it would be better if we don't introduce a new one and just use this value in the `app-bar.tsx` file.

Reply:
@CarlosCortizasCT thanks! Good point - [61753a4](https://github.com/commercetools/merchant-center-application-kit/pull/3288/commits/61753a4bcea2795f2c037df3440eeb32b75ff753)